traitsui/tests/editors/test_tree_editor.py

```diff
@@ -49,6 +51,12 @@ def _wrapped_bogus_default(self):
         return BogusWrap(bogus_list=[Bogus()])
 
 
+class BogusHandler(Handler):
+    def object_expand_all_changed(self, info):
```

Comment:
It took me a moment to figure out the fact that this is a special method - hooked up to listen to the `expand_all` button trait on the `object` of this handler. Is that correct?

Reply:
Yep, see this section of the docs which I used as reference https://docs.enthought.com/traitsui/traitsui_user_manual/handler.html#reacting-to-trait-changes

But magically named methods are generally bad so maybe I should avoid doing this... We need the `UIInfo` object though and the inner magic makes it so these magically named methods are passed it as an argument. (I should dig into the code that does this more, I was just going off the docs)